script/signUp.js

```diff
@@ -165,16 +195,24 @@ function eyeClick_2(e) {
   e.preventDefault();
 }
 
+//accessToken이 있는지 확인하는 함수입니다.
+function checkAccessTokenOnLoginPage() {
```

Comment:
```suggestion
function checkAccessTokenAndRedirect() {
```
좀더 직관적인것 같은데 어떠신가요??